addons/binding/org.openhab.binding.regoheatpump/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Only the newer master contains them here, check out:

https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.airquality/about.html
https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.airquality/build.properties